--- OpenImageIO-oiio-4070df8/src/include/OpenImageIO/SHA1.h.orig	2015-12-26 10:27:30.663439300 +0100
+++ OpenImageIO-oiio-4070df8/src/include/OpenImageIO/SHA1.h	2015-12-26 10:32:29.409487767 +0100
@@ -103,6 +103,8 @@
 #include <stdlib.h>
 #endif
 
+#include <cinttypes>
+
 // You can define the endian mode in your files without modifying the SHA-1
 // source files. Just #define SHA1_LITTLE_ENDIAN or #define SHA1_BIG_ENDIAN
 // in your files, before including the SHA1.h header file. If you don't
@@ -156,7 +158,7 @@
 #ifdef _MSC_VER // Compiling with Microsoft compiler
 #define UINT_8  unsigned __int8
 #else // !_MSC_VER
-#define UINT_8 unsigned char
+#define UINT_8 uint8_t
 #endif // _MSC_VER
 #endif
 
@@ -164,11 +166,7 @@
 #ifdef _MSC_VER // Compiling with Microsoft compiler
 #define UINT_32 unsigned __int32
 #else // !_MSC_VER
-#if (ULONG_MAX == 0xFFFFFFFF)
-#define UINT_32 unsigned long
-#else
-#define UINT_32 unsigned int
-#endif
+#define UINT_32 uint32_t
 #endif // _MSC_VER
 #endif // UINT_32
 
@@ -176,7 +174,7 @@
 #ifdef _MSC_VER // Compiling with Microsoft compiler
 #define INT_64 __int64
 #else // !_MSC_VER
-#define INT_64 long long
+#define INT_64 int64_t
 #endif // _MSC_VER
 #endif // INT_64
 
@@ -184,7 +182,7 @@
 #ifdef _MSC_VER // Compiling with Microsoft compiler
 #define UINT_64 unsigned __int64
 #else // !_MSC_VER
-#define UINT_64 unsigned long long
+#define UINT_64 uint64_t
 #endif // _MSC_VER
 #endif // UINT_64
 
