--- OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp.orig	2013-07-14 17:33:27.426142801 +0200
+++ OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp	2013-07-14 18:10:42.519382338 +0200
@@ -467,7 +467,7 @@
         m_spec.attribute ("dpx:FilmEdgeCode", buf);
 
     tmpstr.clear ();
-    switch (m_dpx.header.Signal ()) {
+    switch ((int)m_dpx.header.Signal ()) {
         case dpx::kUndefined:
             tmpstr = "Undefined";
             break;
--- OpenImageIO-oiio-bcdad81/src/libOpenImageIO/imagebufalgo.cpp~	2013-11-01 18:03:46.000000000 +0100
+++ OpenImageIO-oiio-bcdad81/src/libOpenImageIO/imagebufalgo.cpp	2015-02-18 17:15:01.274966027 +0100
@@ -1063,9 +1063,9 @@
         if (error)
             continue;  // ignore errors
         // now, draw to our target surface
-        for (int j = 0;  j < slot->bitmap.rows; ++j) {
+        for (unsigned int j = 0;  j < slot->bitmap.rows; ++j) {
             int ry = y + j - slot->bitmap_top;
-            for (int i = 0;  i < slot->bitmap.width; ++i) {
+            for (unsigned int i = 0;  i < slot->bitmap.width; ++i) {
                 int rx = x + i + slot->bitmap_left;
                 float b = slot->bitmap.buffer[slot->bitmap.pitch*j+i] / 255.0f;
                 R.getpixel (rx, ry, pixelcolor);
