--- OpenImageIO-2.4.17.0/src/cineon.imageio/CMakeLists.txt.orig	2024-04-26 19:25:33.286182160 +0200
+++ OpenImageIO-2.4.17.0/src/cineon.imageio/CMakeLists.txt	2024-04-26 19:49:10.735169846 +0200
@@ -3,9 +3,7 @@
 # https://github.com/AcademySoftwareFoundation/OpenImageIO
 
 add_oiio_plugin (cineoninput.cpp
-                 libcineon/Cineon.cpp libcineon/OutStream.cpp libcineon/Codec.cpp
-                 libcineon/Reader.cpp libcineon/CineonHeader.cpp
-                 libcineon/ElementReadStream.cpp libcineon/InStream.cpp)
+                 LINK_LIBRARIES cineon)
 
 # Note: OIIO doesn't support cineon output, so we don't compile
 # libcineon/Writer.cpp
--- oiio-2.4.14.0/src/cineon.imageio/cineoninput.cpp.orig	2023-08-01 22:15:24.000000000 +0200
+++ oiio-2.4.14.0/src/cineon.imageio/cineoninput.cpp	2023-08-22 17:46:26.265122755 +0200
@@ -4,7 +4,7 @@
 
 #include <cmath>
 
-#include "libcineon/Cineon.h"
+#include <Cineon.h>
 
 #include <OpenImageIO/dassert.h>
 #include <OpenImageIO/imageio.h>
@@ -337,8 +337,8 @@ CineonInput::open(const std::string& nam
         // FIXME: do something about the time zone
     }
     {
-        char filmedge[17];
-        m_cin.header.FilmEdgeCode(filmedge, sizeof(filmedge));
+        char filmedge[3+3+3+10+10+1]; // should be 17 (2+2+2+6+4+NUL), but be safe in case of out-of-range values
+        m_cin.header.FilmEdgeCode(filmedge);
         if (filmedge[0])
             m_spec.attribute("cineon:FilmEdgeCode", filmedge);
     }
