--- OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp.orig	2013-07-14 17:33:27.426142801 +0200
+++ OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp	2013-07-14 18:10:42.519382338 +0200
@@ -467,7 +467,7 @@
         m_spec.attribute ("dpx:FilmEdgeCode", buf);
 
     tmpstr.clear ();
-    switch (m_dpx.header.Signal ()) {
+    switch ((int)m_dpx.header.Signal ()) {
         case dpx::kUndefined:
             tmpstr = "Undefined";
             break;

--- OpenImageIO-oiio-4070df8/src/libtexture/texturesys.cpp~	2015-12-19 07:44:03.000000000 +0100
+++ OpenImageIO-oiio-4070df8/src/libtexture/texturesys.cpp	2015-12-26 10:38:06.735886871 +0100
@@ -2038,7 +2038,7 @@
                     int pixelsize = tile->pixelsize();
                     int offset = pixelsize * (tile_t * spec.tile_width + tile_s);
                     offset += (firstchannel - id.chbegin()) * channelsize;
-                    DASSERT ((size_t)offset < spec.tile_width*spec.tile_height*spec.tile_depth*pixelsize);
+                    DASSERT ((size_t)offset < (size_t)spec.tile_width*spec.tile_height*spec.tile_depth*pixelsize);
                     if (pixeltype == TypeDesc::UINT8)
                         texel_simd[j][i] = uchar2float4 ((const unsigned char *)(tile->bytedata() + offset));
                     else if (pixeltype == TypeDesc::UINT16)
