--- OpenImageIO-oiio-7d98ca6/src/dpx.imageio/CMakeLists.txt.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dpx.imageio/CMakeLists.txt	2011-11-26 09:02:29.701499722 +0100
@@ -1,5 +1,3 @@
 add_oiio_plugin (dpxinput.cpp dpxoutput.cpp
-  libdpx/DPX.cpp libdpx/OutStream.cpp libdpx/RunLengthEncoding.cpp
-  libdpx/Codec.cpp libdpx/Reader.cpp libdpx/Writer.cpp libdpx/DPXHeader.cpp
-  libdpx/ElementReadStream.cpp libdpx/InStream.cpp libdpx/DPXColorConverter.cpp
-  LINK_LIBRARIES ${OPENEXR_LIBRARIES})
+  libdpx/DPXColorConverter.cpp
+  LINK_LIBRARIES dpx)
--- OpenImageIO-oiio-7d98ca6/src/dpx.imageio/dpxinput.cpp.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dpx.imageio/dpxinput.cpp	2011-11-26 09:03:24.121501545 +0100
@@ -28,7 +28,7 @@
   (This is the Modified BSD License)
 */
 
-#include "libdpx/DPX.h"
+#include <DPX.h>
 #include "libdpx/DPXColorConverter.h"
 
 #include "typedesc.h"
@@ -499,7 +499,7 @@
         case dpx::k1125LineProgressive169AR:
             tmpstr = "YCbCr 1125p, 16:9 (SMPTE 274M)";
             break;
-        case dpx::k255:
+        case 255:
             // don't set the attribute at all
             break;
         default:
--- OpenImageIO-oiio-7d98ca6/src/dpx.imageio/dpxoutput.cpp.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dpx.imageio/dpxoutput.cpp	2011-11-26 09:03:35.934835274 +0100
@@ -33,7 +33,7 @@
 #include <cmath>
 #include <boost/algorithm/string.hpp>
 
-#include "libdpx/DPX.h"
+#include <DPX.h>
 #include "libdpx/DPXColorConverter.h"
 
 #include "typedesc.h"
--- OpenImageIO-oiio-7d98ca6/src/dpx.imageio/libdpx/DPXColorConverter.h.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dpx.imageio/libdpx/DPXColorConverter.h	2011-11-26 09:05:17.444838673 +0100
@@ -37,7 +37,7 @@
 #define _DPX_COLORCONVERTER_H 1
 
 
-#include "DPX.h"
+#include <DPX.h>
 
 
 namespace dpx
