--- OpenImageIO-oiio-7d98ca6/src/ptex.imageio/CMakeLists.txt.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/ptex.imageio/CMakeLists.txt	2011-11-19 14:09:32.265198985 +0100
@@ -1,13 +1,3 @@
 add_oiio_plugin (ptexinput.cpp ptexoutput.cpp
-                 ptex/PtexCache.cpp
-                 ptex/PtexFilters.cpp
-                 ptex/PtexHalf.cpp
-                 ptex/PtexReader.cpp
-                 ptex/PtexSeparableFilter.cpp
-                 ptex/PtexSeparableKernel.cpp
-                 ptex/PtexTriangleFilter.cpp
-                 ptex/PtexTriangleKernel.cpp
-                 ptex/PtexUtils.cpp
-                 ptex/PtexWriter.cpp
-                 )
+	LINK_LIBRARIES Ptex)
 
--- OpenImageIO-oiio-7d98ca6/src/ptex.imageio/ptexoutput.cpp.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/ptex.imageio/ptexoutput.cpp	2011-11-19 14:19:10.445218345 +0100
@@ -28,7 +28,7 @@
   (This is the Modified BSD License)
 */
 
-#include "ptex/PtexWriter.h"
+#include <Ptexture.h>
 
 #include "typedesc.h"
 #include "imageio.h"
--- OpenImageIO-oiio-7d98ca6/src/ptex.imageio/ptexinput.cpp.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/ptex.imageio/ptexinput.cpp	2011-11-19 14:19:35.705219191 +0100
@@ -28,7 +28,7 @@
   (This is the Modified BSD License)
 */
 
-#include "ptex/Ptexture.h"
+#include <Ptexture.h>
 
 #include "dassert.h"
 #include "typedesc.h"
