Field3D uses only C interface of hdf5
--- OpenImageIO-oiio-f4d79c0/src/cmake/externalpackages.cmake.orig	2013-07-14 17:08:40.729538530 +0200
+++ OpenImageIO-oiio-f4d79c0/src/cmake/externalpackages.cmake	2013-07-14 17:09:33.429536318 +0200
@@ -284,7 +284,7 @@
         # N.B. For a custom version, the caller had better set up the
         # variables HDF5_INCLUDE_DIRS and HDF5_LIBRARIES.
     else ()
-        find_package (HDF5 COMPONENTS CXX)
+        find_package (HDF5)
     endif ()
     if (VERBOSE)
         message (STATUS "HDF5_FOUND=${HDF5_FOUND}")
