--- OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp.orig	2013-07-14 17:33:27.426142801 +0200
+++ OpenImageIO-oiio-f4d79c0/src/dpx.imageio/dpxinput.cpp	2013-07-14 18:10:42.519382338 +0200
@@ -467,7 +467,7 @@
         m_spec.attribute ("dpx:FilmEdgeCode", buf);
 
     tmpstr.clear ();
-    switch (m_dpx.header.Signal ()) {
+    switch ((int)m_dpx.header.Signal ()) {
         case dpx::kUndefined:
             tmpstr = "Undefined";
             break;
