--- OpenImageIO-oiio-f4d79c0/src/libOpenImageIO/CMakeLists.txt.orig	2013-07-14 16:40:55.296275085 +0200
+++ OpenImageIO-oiio-f4d79c0/src/libOpenImageIO/CMakeLists.txt	2013-07-14 17:08:13.526206338 +0200
@@ -210,7 +210,7 @@
 endif ()
 target_link_libraries (OpenImageIO
                            ${VISIBILITY_COMMAND} ${VISIBILITY_MAP_COMMAND}
-                           ${Boost_LIBRARIES})
+			   ${Boost_LIBRARIES} ${CMAKE_DL_LIBS})
 
 # Link against system TBB library if specified
 if (USE_TBB AND USE_EXTERNAL_TBB)
