--- OpenImageIO-oiio-7d98ca6/src/dds.imageio/CMakeLists.txt.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dds.imageio/CMakeLists.txt	2011-11-18 21:22:02.993174921 +0100
@@ -1 +1 @@
-add_oiio_plugin (ddsinput.cpp ddsoutput.cpp squish/alpha.cpp squish/clusterfit.cpp squish/colourblock.cpp squish/colourfit.cpp squish/colourset.cpp squish/maths.cpp squish/rangefit.cpp squish/singlecolourfit.cpp squish/squish.cpp)
+add_oiio_plugin (ddsinput.cpp ddsoutput.cpp LINK_LIBRARIES squish)
--- OpenImageIO-oiio-7d98ca6/src/dds.imageio/ddsinput.cpp.orig	2011-11-05 08:35:00.000000000 +0100
+++ OpenImageIO-oiio-7d98ca6/src/dds.imageio/ddsinput.cpp	2011-11-18 21:30:04.823191054 +0100
@@ -39,7 +39,7 @@
 #include "imageio.h"
 #include "fmath.h"
 
-#include "squish/squish.h"
+#include <squish.h>
 
 OIIO_PLUGIN_NAMESPACE_BEGIN
 
