diff -urNp -x '*.orig' oiio-Release-2.0.13.org/src/dds.imageio/CMakeLists.txt oiio-Release-2.0.13/src/dds.imageio/CMakeLists.txt
--- oiio-Release-2.0.13.org/src/dds.imageio/CMakeLists.txt	2019-12-03 23:28:14.000000000 +0100
+++ oiio-Release-2.0.13/src/dds.imageio/CMakeLists.txt	2021-03-07 10:21:59.270402883 +0100
@@ -1,4 +1 @@
-add_oiio_plugin (ddsinput.cpp squish/alpha.cpp squish/clusterfit.cpp
-                 squish/colourblock.cpp squish/colourfit.cpp squish/colourset.cpp
-                 squish/maths.cpp squish/rangefit.cpp squish/singlecolourfit.cpp
-                 squish/squish.cpp)
+add_oiio_plugin (ddsinput.cpp LINK_LIBRARIES squish)
diff -urNp -x '*.orig' oiio-Release-2.0.13.org/src/dds.imageio/ddsinput.cpp oiio-Release-2.0.13/src/dds.imageio/ddsinput.cpp
--- oiio-Release-2.0.13.org/src/dds.imageio/ddsinput.cpp	2019-12-03 23:28:14.000000000 +0100
+++ oiio-Release-2.0.13/src/dds.imageio/ddsinput.cpp	2021-03-07 10:21:59.273736603 +0100
@@ -39,7 +39,7 @@
 #include <OpenImageIO/imageio.h>
 #include <OpenImageIO/typedesc.h>
 
-#include "squish/squish.h"
+#include <squish.h>
 
 OIIO_PLUGIN_NAMESPACE_BEGIN
 
