--- OpenImageIO-oiio-4070df8/src/libutil/CMakeLists.txt~	2015-12-19 07:44:03.000000000 +0100
+++ OpenImageIO-oiio-4070df8/src/libutil/CMakeLists.txt	2015-12-26 10:54:02.944577313 +0100
@@ -16,6 +16,7 @@
     target_link_libraries (OpenImageIO_Util psapi.lib)
 endif ()
 target_link_libraries (OpenImageIO_Util ${ILMBASE_LIBRARIES})
+target_link_libraries (OpenImageIO_Util ${CMAKE_DL_LIBS})
 
 if (VERBOSE)
     message(STATUS "Setting SOVERSION to: ${SOVERSION}")
